frontend/src/api/metrics/getTopLevelOperations.ts

```diff
@@ -1,7 +1,19 @@
 import axios from 'api';
 
-const getTopLevelOperations = async (): Promise<ServiceDataProps> => {
-	const response = await axios.post(`/service/top_level_operations`);
+interface GetTopLevelOperationsProps {
+	service?: string;
+	start?: number;
+	end?: number;
```

Comment:
Can they ever be undefined? If so, it going to be problematic

Reply:
no they cannot be, just added safety checks here as in the second case we didn't have the service so handled such cases. but they will not be undefined